.grit/patterns/go/go_cloudflare_v5.md

```diff
@@ -2,13 +2,13 @@
 title: Upgrade to CloudFlare v5
 ---
 
-The Go SDK has been rewritten for v2 and contains significant breaking changes.
+The Go SDK has been rewritten for v5 and contains significant breaking changes.
```

Comment:
It's v2: https://github.com/cloudflare/cloudflare-go/tree/v2

Reply:
No? It's [v1](https://github.com/cloudflare/cloudflare-go) to [v2](https://github.com/cloudflare/cloudflare-go/tree/v2). I messed up the original title.